templates/arbiter_outputs_entry.py

```diff
@@ -0,0 +1,247 @@
+        self.zfill = len(str(self.ff_end)) + 1
+        self.loop_count = 0
+
+        self.next_counter_soon()
+
+    def next_counter_soon(self):
+        self.setup_counter_timer = QTimer()
+        self.setup_counter_timer.singleShot(1, self.setup_next_counter)
+
+    def setup_next_counter(self):
+        fault_num_str = str(self.fault_num).zfill(self.zfill)
+        # Use the bypass channel to get the bypass counts
+        bypass_counter = CounterElement(
+            pvname=f"ca://{self.prefix}FFO:{self.ffo}:FF:{fault_num_str}:Ovrd:Active_RBV",
+            index=self.loop_count,
+            value_cache=self.bypasses,
+            parent=self,
+        )
+        bypass_dest = PyDMChannel(
+            address=f"loc://{self.prefix}{self.ffo}:BypassCount?type=int&init=0",
+            value_slot=self.new_bypass_count,
+            value_signal=bypass_counter.value_sig,
+        )
+        self.bypass_counters.append(bypass_counter)
+        self.bypasses.append(0)
+        bypass_dest.connect()
+        self._channels.append(bypass_dest)
+        bypass_ch = bypass_counter.create_channel()
+        bypass_ch.connect()
+        self._channels.append(bypass_ch)
+
+        # Use the in_use channel to get the registered and connected counts
+        in_use_counter = CounterElement(
+            pvname=f"ca://{self.prefix}FFO:{self.ffo}:FF:{fault_num_str}:Info:InUse_RBV",
+            index=self.loop_count,
+            value_cache=self.in_uses,
+            conn_cache=self.is_connected,
+            parent=self,
+        )
+        in_use_dest = PyDMChannel(
+            address=f"loc://{self.prefix}{self.ffo}:RegCount?type=int&init=0",
+            value_slot=self.show_loc_connected,
+            value_signal=in_use_counter.value_sig,
+        )
+        in_use_conn_dest = PyDMChannel(
+            address=f"loc://{self.prefix}{self.ffo}:ConnCount?type=int&init=0",
+            value_slot=self.show_loc_connected,
+            value_signal=in_use_counter.conn_sig,
+        )
+        self.in_use_counters.append(in_use_counter)
+        self.in_uses.append(0)
+        self.is_connected.append(0)
+        in_use_dest.connect()
+        in_use_conn_dest.connect()
+        self._channels.append(in_use_dest)
+        self._channels.append(in_use_conn_dest)
+        ic_ch = in_use_counter.create_channel()
+        ic_ch.connect()
+        self._channels.append(ic_ch)
+
+        # Combine the ok and in_use channels to get the fault counts
+        # We are fauling if ok=False and in_use=True
+        # Summarize this as a combined local channel
+        fault_summary = FaultSummary(
+            ok_pvname=f"ca://{self.prefix}FFO:{self.ffo}:FF:{fault_num_str}:OK_RBV",
+            in_use_pvname=f"ca://{self.prefix}FFO:{self.ffo}:FF:{fault_num_str}:Info:InUse_RBV",
+            parent=self,
+        )
+        fault_summary_ch1, fault_summary_ch2 = fault_summary.create_channels()
+        fault_summary_ch3 = PyDMChannel(
```

Comment:
I was a little surprised that this and `fault_counter` weren't pat of `FaultSummary`. I imagine it just felt cleaner to keep as-is?

Reply:
I'll give this another look

Reply:
This is 100% correct, this must be because I added `FaultSummary` as an afterthought and kept the same structure I used for the other counters. I'll revise appropriately for clarity along with some other clarity revisions (e.g. stop saying pvname for addresses that can be loc://)